src/sage/sets/set.py

```diff
@@ -602,6 +615,19 @@ def _an_element_(self):
                 pass
         return self._an_element_from_iterator()
 
+    def random_element(self):
+        """
+        Return a random element of ``self``.
+
+        EXAMPLES::
+
+            sage: Set(ZZ).random_element()  # random
+            4
```

Comment:
```suggestion
            sage: Set(ZZ).random_element()  in ZZ
            True
```
Best to test the properties.